scrapinghub/client.py

```diff
@@ -203,6 +195,17 @@ def _extract_spider_id(self, params):
             raise ValueError('Please use same spider id')
         return spiderid
 
+    def update_tags(self, add=None, remove=None, spidername=None):
+        spidername = spidername or (self.spider.name if self.spider else None)
+        if not spidername:
+            raise ValueError('Please provide spidername')
```

Comment:
@vshlapakov this validation check starts worrying me, shouldn't it be done on the dash side?